src/utils.jl

```diff
@@ -0,0 +1,181 @@
+# When a specific branch is loaded via `]add Package#branch` julia will only
```

Comment:
Maybe add even more utility stuff from `Oscar.jl` like `test_module`, etc.? What do you think?

Reply:
Done.